src/selectionDOMRenderer.js

```diff
@@ -214,7 +214,7 @@ var SelectionDOMRenderer = function(options, state) {
     render(state);
   }
 
-  render(state);
```

Comment:
The renderer should never have any externally accessible methods. This encourages state to be stored there. Obviously exposing this method doesn't set anything, but a rendering should be a direct and always up-to-date projection of the state of the thing it renders. The renderer should call its own render method when the event emitter emits the update event. This makes it possible to feed multiple consumers through the component's outer API (like jCrop).

I will finish rigging this up tonight, so it will be easier to proceed. 

For more background, if you have time, check out: 
[flux application/component architecture](https://facebook.github.io/flux/docs/overview.html) (watch at 2X speed or read the quick summary)
[Nicholas Hery - Describing UI State with Data](http://nicolashery.com/describing-ui-state-with-data/) (ignore the stupid hat)
[cycleJS explicit data flow](http://cycle.js.org/)
[Time-traveling debugger allows you to _actually see_ what's happening](http://cycle.js.org/cycle-time-travel/)